apps/src/templates/teacherDashboard/RosterDialog.jsx

```diff
@@ -163,11 +164,51 @@ class RosterDialog extends React.Component {
       classrooms.find(classroom => {
         return classroom.id === this.state.selectedId;
       }).name;
-
     this.props.handleImport(this.state.selectedId, selectedName);
     this.setState({selectedId: null});
   };
 
+  // create new function for redirect to NewEditPage
+  redirectToEditSectionPage = sectionId => {
+    const redirectUrl = '/sections/' + sectionId + '/edit';
+    window.location.href = redirectUrl;
+  };
+
+  // Need to update with section ID
```

Comment:
We got that section ID! Might still be helpful to leave a comment here with a quick description of the functionality!